### DISCUSSION TITLE
Default AWS parameter store paths

### DISCUSSION THREAD

**Discussion by aidanhmiles** [mozilla/hubs#2863] [General Questions & Discussion] [answered]
Hello,
I had a working self-hosted Hub, then I messed up CORS, then I deleted the CORS origin parameter from the AWS parameter store, but new instances are failing to connect to Reticulum (I think) due to this error:

Content Security Policy: Couldn’t parse invalid host wss://:80
Content Security Policy: Couldn’t parse invalid host https://:80

I'd like to try restoring previously working connect-src and CORS origin parameters into the store, but I can't find references to the path for connect-src (and I'm only half certain I found the right one for CORS, at /ita/{hubname}/reticulum/security/cors_origins). 

Can anyone share the AWS parameter store path for cors and connect-src? Thanks in advance.

**Comment by robinkwilson** [comment, marked as answer]
The one parameter store param is `/ita/{stack name}/reticulum/security/cors_origins`.
Deleting the setting in the parameter store should be enough, but you can also restart the instance: 

AWS console > EC2 > click on your HC server(s) then "Terminate" instances. New ones will start up immediately after.

Tell me how it goes.

**Reply by robinkwilson** [reply]
Here's cors settings that work for me : "https://{ main domain }.com, https://{ link domain }.link, { main domain }.com, { link domain }.link"

**Comment by aidanhmiles** [comment]
Thanks for the reply. I think CORS is getting me part of the way, and I can create / operate Rooms just fine, and Spoke appears to load and work. The admin interface, however, loads as a white screen. I'm not sure if this is actually a different issue that I don't recognize. Searching for white page / blank page / admin panel issues so far hasn't turned up an answer.

**Reply by aidanhmiles** [reply]
Ok, I think I'm just not used to the flow of "make config change, rollover instances", because I've removed the configs from the parameter store, replaced them, changed them, and made new instances between and nothing has changed. I think this is probably not related to CORS or CSP. I'll open a new ticket. Thanks for your help.

**Reply by robinkwilson** [reply]
Continued in https://github.com/mozilla/hubs/discussions/2866
